README.md

```diff
@@ -171,6 +171,23 @@ On openSUSE, install the [`exa`](https://software.opensuse.org/package/exa) pack
 On Ubuntu 20.10 (Groovy Gorilla) and later, install the [`exa`](https://packages.ubuntu.com/groovy/exa) package.
 
     $ sudo apt install exa
+    
+On Ubuntu 20.04 (Focal Fossa) and earlier, make and change into a temp directory 
+    $ mkdir temp && cd temp 
+
+Download the zip file and extract the files.
+    $ wget https://github.com/ogham/exa/releases/download/v0.10.0/exa-linux-x86_64-v0.10.0.zip && unzip exa-linux-x86_64-v0.10.0.zip
+
+Move the files to the required directories.
+    
+    $sudo mv  bin/exa  /usr/bin/ && sudo mv  man/exa/* /usr/share/man/man1 && sudo mv exa/exa/completions/* /etc/bash_completion.d
+    
+    $sudo mv /etc/bash_completion.d/completions.fish  /usr/share/fish/vendor_completions.d/
+    
+    $sudo mv /etc/bash_completion.d/completions.zsh  /usr/share/zsh/vendor-completions/
+
+    
```

Comment:
- Use `cp` since it’s more usual.
- There are two man pages now
- completions should be named differently depending on the shell
- `site-functions` is the appropriate directory nowadays for zsh completions

```suggestion
Copy the files to the required directories:
    
    $ sudo cp bin/exa /usr/bin/
    $ sudo cp man/exa.1 /usr/share/man/man1 && sudo cp man/exa_colors.5 /usr/share/man/man5
    $ sudo cp completions/completions.bash /etc/bash_completion.d/exa
    $ sudo cp completions/completions.fish /usr/share/fish/vendor_completions.d/exa.fish
    $ sudo cp completions/completions.zsh /usr/share/zsh/site-functions/_exa
```